woocommerce/woocommerce-gateway-stripe
# Hide PRB box in checkout block when PRBs disabled

Fixes #2336

## Changes proposed in this Pull Request:

This simply adds an additional check if the `payment_request` Stripe settings isn't "yes" and returns false so that the PRB box is not shown in the checkout block.

## Testing instructions

- Uncheck "Enable express checkouts" in the Stripe settings and save.
- Add something to your cart and go to a page with the checkout block.
- Before the fix, the PRB box would render (but be empty). After the fix, the PRB box should not be shown if "Enable express checkouts" is not checked.

---


**Post merge**
